### DISCUSSION TITLE
Filter Charts

### DISCUSSION THREAD

**Discussion by cyber-vi-king** [firefly-iii/firefly-iii#8338] [Firefly III]
Hi, is it possible to filter charts? 

I have a budget called Bills and while it is incredibly useful to tag stuff with that I'd like to hide it from the budgets and spending chart to see the other budgets clearer (as I can't do that much about spendings regarding bills anyways)

Same goes for the categories. I try to categorise everything, so I have a category transfers (for moving money between accounts) and I'd like to hide that one along with "no category" and a category related to bills from the chart to see the others more clearly. 

I think this could be solved with "enabled in charts" checkboxes in the respective settings (category, budget, ...)

But I'd also be happy with some DIY solutions involving custom_css and custom_js. I just haven't found how. 

Hope this makes sense. Looking forward to hearing ideas about this.

**Comment by JC5** [comment]
This isn't possible right now, the dashboard shows everything. You can fiddle around with the `/reports` though.
